libbrowser/libbrowser.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,432 @@
+// !$*UTF8*$!
```

Comment:
The project file should be generated by Gyp and shouldn't be part of the source tree.